micromasters/serializers_test.py

```diff
@@ -0,0 +1,50 @@
+"""
+Tests for serializing Django User objects
+"""
+from django.contrib.auth.models import AnonymousUser
+from django.db.models.signals import post_save
+from factory.django import mute_signals
+from micromasters.serializers import UserSerializer
+from micromasters.factories import UserFactory
+from profiles.factories import ProfileFactory
+from search.base import ESTestCase
+# pylint: disable=no-self-use
+
+
+class UserTests(ESTestCase):
+    "User tests"
```

Comment:
All docstrings should be surrounded by three double quotes: https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring